src/v2i-hub/TelematicBridgePlugin/manifest.json

```diff
@@ -11,6 +11,31 @@
             "key": "LogLevel",
             "default": "INFO",
             "description": "The log level for this plugin"
+        },
+        {
+            "key": "NATSUrl",
+            "default": "nats://127.0.0.1:4222",
+            "description": "The NATS connection URL"            
+        },
+        {
+            "key": "UnitId",
+            "default": "v2xhub_id",
+            "description": "The unique identifier for a telematic unit"            
+        },
+        {
+            "key": "UnitName",
+            "default": "East Intersection",
```

Comment:
Can this match J2735 intersection name or does it need to be separate. If it can match I suggest we just add in the description to set this to intersection name

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/567/commits/1c59a3e2d0a95ea4ab44bc691b001aad589bd19c

Reply:
Please specify J2735 MAP in the description. So should be set to MAP J2735 intersection name if available.

Reply:
This is not specific about MAP message or intersection. If we deploy this v2xhub in Cav-in-a-box at roadside, there is no concept of intersection. This east intersection is just a default value I give cause I test it with east intersection.

Reply:
I know this is not MAP Message specific from an implementation stand point but we should not be creating unnecessary new IDs in our different pieces of functionality. For example `INFRASTRUCTURE_ID` is a unique string used for registration for simulation but if we can reuse the ID for say telematics or if we could relate that ID to the unique J2735 Intersection ID we decrease the number of separately managed ID parameters. This way they all relate to each other.

Reply:
Can we table this for now? The unit id is something we used in the telematic tool context. The same design for this unit id is applied to carma-streets, carma-platform (vehicle), and carma-cloud. If this has some conflicts with IDs in a simulated environment, we probably want to decide and make the changes to all units to better fit the simulated environment.

Reply:
Do we have unique IDs for carma streets, carma-cloud , carma-platform as well to register for the simulation?